src/record.ts

```diff
@@ -44,6 +45,7 @@ class Record<TFields extends FieldSet> {
     constructor(table: Table<TFields>, recordId: string, recordJson?: RecordJson) {
         this._table = table;
         this.id = recordId || recordJson.id;
+        this.commentCount = recordId || recordJson.commentCount;
```

Comment:
This will set `commentCount` to the value of `recordId` if it is truthy (probably not what we want). Suggestion:

```suggestion
        if (recordJson) {
          this.commentCount = recordJson.commentCount;
        }
```